libs/common/src/auth/abstractions/devices/responses/device.response.ts

```diff
@@ -15,6 +15,9 @@ export class DeviceResponse extends BaseResponse {
   creationDate: string;
   revisionDate: string;
   isTrusted: boolean;
+  encryptedUserKey: string;
+  encryptedPublicKey: string;
```

Comment:
```suggestion
  encryptedUserKey: string | null;
  encryptedPublicKey: string | null;
```

:pick: I would recommend making it clear these are expected to be null (untrusted devices).